runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierConfig.java

```diff
@@ -110,10 +111,36 @@ default int getPrayerThreshold()
 		return 0;
 	}
 
+	@ConfigItem(
+		keyName = "lowEnergy",
+		name = "Low Energy Notification Threshold",
+		description = "The amount of energy points remaining to send a notification at. A value of 0 will disable notification.",
```

Comment:
Wouldn't users want to be able to be notified when they run out of energy completely?

Reply:
I'll set this one to not trigger when the value is set to 100 then. I had it like this because I wanted it to be consistent with the other notifications, but I agree this is better.